cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterControllerConfig.java

```diff
@@ -3,22 +3,60 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Cluster Controller configuration
+ */
 public class ClusterControllerConfig {
 
     public static final String STRIMZI_NAMESPACE = "STRIMZI_NAMESPACE";
     public static final String STRIMZI_CONFIGMAP_LABELS = "STRIMZI_CONFIGMAP_LABELS";
+    public static final String STRIMZI_FULL_RECONCILIATION_INTERVAL = "STRIMZI_FULL_RECONCILIATION_INTERVAL";
+
+    private static final long DEFAULT_FULL_RECONCILIATION_INTERVAL = 120_000; // in ms (2 minutes)
 
     private Map<String, String> labels;
     private String namespace;
+    private long reconciliationInterval;
 
-    public ClusterControllerConfig(String namespace, Map<String, String> labels) {
+    /**
+     * Constructor
```

Comment:
We need to come to a consensus on javadoc. For me, the doc "Constructor" on a constructor adds no value: I already know it's a constructor! Imho just the docs on the parameters would be sufficient here. I raise this not because I want you to change it, but because I just want us all to be doing the same thing.

Reply:
I agree we should define some common rules for this kind of things.